README.md

```diff
@@ -307,6 +292,56 @@ The collector also serves as a control plane for agents/clients by supplying
 them updated configuration (e.g. trace sampling policies), and reporting
 agent/client health information/inventory metadata to downstream exporters.
 
+### <a name="global-tags"></a> Global Tags
+
+The collector also takes some global configurations that modify its behavior for all receivers / exporters. One of the configurations
+available is to add Attributes or Tags to all spans passing through this collector. These additional tags can be configured to either overwrite
+attributes if they already exists on the span, or respect the original values. An example of this is provided below.
+```yaml
+global:
+  attributes:
+    overwrite: true
+    values:
+      # values are key value pairs where the value can be an int, float, bool, or string
+      some_string: "hello world"
+      some_int: 1234
+      some_float: 3.14159
+      some_bool: false
+```
+
+### <a name="tail-sampling"></a>Intelligent Sampling
+
+```yaml
+sampling:
+  mode: tail
+  # amount of time from seeing the first span in a trace until making the sampling decision
+  decision-wait: 10s
+  # maximum number of traces kept in the memory
+  num-traces: 500
+  policies:
+    # user-defined policy name
+    my-rate-limiting:
+      # exporters the policy applies to
+      exporters:
+        - jaeger
+        - omnition
+      policy: rate-limiting
+      configuration:
+        spans-per-second: 10
+#      policy: string-tag-filter
```

Comment:
I think these here as a comment are confusing. Either we added user defined policies and use them or do not include them.